File: JavaCertification_1Z0-808/src/mdc/objects/Person.java
package mdc.learningjava;

/**
 * @author Dunieski 10/06/2020
 * Class Structure
 */

public class Person {
    private String firstName; // firstName Person's attribute
    private String lastName; // lastName Person's attribute
    private int age; // age Person's attribute

    public String getFirstName(){
        return firstName;
    }

    public void setFirstName(String firstName){
        this.firstName=firstName;
    }

    public String getLastName(){
        return lastName;
    }

    public void setLastName(String lastName){
        this.lastName=lastName;
    }

    public int getAge(){
        return age;
    }

    public void setAge(int age){
        this.age=age;
    }
}

File: Classes-and-Objects/Person/person.py
class Person:
    def __init__(self, fname =None,lname = None, age = None, gender=None):
        self.fname = fname
        self.lname = lname
        self.age = age
        self.gender = gender

    
    def setFirstName(self, fname):
        self.fname = fname

    def getFirstName(self):
        return self.fname

    def setLastName(self, lname):
        self.lname = lname

    def getLastName(self):
        return self.lname

    def setAge(self, age):
        self.age = age

    def getAge(self):
        return self.age
    
    def setGender(self, gender):
        self.gender = gender
        
    def getGender(self):
        return self.gender